base/sparse/cholmod.jl

```diff
@@ -1325,4 +1325,7 @@ function ishermitian(A::Sparse{Complex{Float64}})
     end
 end
 
+(*){Ti}(A::Symmetric{Float64,SparseMatrixCSC{Float64,Ti}}, B::SparseMatrixCSC{Float64,Ti}) = sparse(Sparse(A)*Sparse(B))
```

Comment:
I'm open for a discussion about this, but the reasoning is that the namespace takes care of the potential confusion because `Sparse` is not exported.